### PR TITLE
Pv2 2118 project rename

### PR DESCRIPTION
**Pull request by hardik-desai-kcl**

### CODE REVIEW

File: src/SFA.DAS.Payments.MatchedLearner.AcceptanceTests/TestClient.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        private static HttpClient _client = new HttpClient();
+        private readonly string _url;
+
+        public TestClient()
+        {
+            _url = TestConfiguration.TargetUrl;
+
+            if (!string.IsNullOrEmpty(_url)) return;
+
+            _client = new CustomWebApplicationFactory<Startup>().CreateClient(new WebApplicationFactoryClientOptions { BaseAddress = new Uri("https://localhost:44300") });
```
Comment (carlbm) [inline src/SFA.DAS.Payments.MatchedLearner.AcceptanceTests/TestClient.cs:41]: There's no reason to have the options. Will work just fine as: 
`new CustomWebApplicationFactory<Startup>().CreateClient();`

Comment (hardik-desai-kcl) [reply src/SFA.DAS.Payments.MatchedLearner.AcceptanceTests/TestClient.cs:41]: without the option the base Url is not getting set

File: src/SFA.DAS.Payments.MatchedLearner.AcceptanceTests/Bindings/SmokeTestBindings.cs [hunk 2]
```diff
@@ -0,0 +1,153 @@
+﻿using System;
```
[the remark this answers is not in the document]
Comment (hardik-desai-kcl) [reply src/SFA.DAS.Payments.MatchedLearner.AcceptanceTests/Bindings/SmokeTestBindings.cs]: fixed

File: .gitignore [hunk 3]
```diff
@@ -267,24 +267,16 @@ src/SFA.DAS.EAS.Web/App_Data/audit/*.*
...
 
 # Azure Functions localsettings file
 local.settings.json
 
 # Internal nlog log
-**/internal-nlog.txt
\ No newline at end of file
+**/internal-nlog.txt
+
+src/MatchedLearnerApi.AcceptanceTests/appsettings.Development.json
```
[the remark this answers is not in the document]
Comment (hardik-desai-kcl) [reply .gitignore]: fixed

File: src/SFA.DAS.Payments.MatchedLearner.AcceptanceTests/Bindings/SmokeTestBindings.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (hardik-desai-kcl) [reply src/SFA.DAS.Payments.MatchedLearner.AcceptanceTests/Bindings/SmokeTestBindings.cs]: SFA.DAS.Payments.MatchedLearner.AcceptanceTests nameSpace is correct, I am not sure what you are suggesting

[the remark this answers is not in the document]
Comment (carlbm) [reply src/SFA.DAS.Payments.MatchedLearner.AcceptanceTests/Bindings/SmokeTestBindings.cs]: That these are acceptance tests for the API. If we add something that is not an API and has acceptance tests then what will we call them?

File: src/SFA.DAS.Payments.MatchedLearner.AcceptanceTests/TestClient.cs (hunk 1 above)
Comment (carlbm) [reply src/SFA.DAS.Payments.MatchedLearner.AcceptanceTests/TestClient.cs:41]: No need for it to be set. It works fine without it

Comment (hardik-desai-kcl) [reply src/SFA.DAS.Payments.MatchedLearner.AcceptanceTests/TestClient.cs]: removed

File: src/SFA.DAS.Payments.MatchedLearner.AcceptanceTests/Bindings/SmokeTestBindings.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (hardik-desai-kcl) [reply src/SFA.DAS.Payments.MatchedLearner.AcceptanceTests/Bindings/SmokeTestBindings.cs]: I think we can solve that problem when we get there, for now its testing whole of MatchedLearner Domain so I would like to leave it as it

**Review by carlbm — APPROVED** [review]

**Review by carlbm — APPROVED** [review]

**Pull request merged**